defaults/main.yml

```diff
@@ -34,6 +34,7 @@ acme_sh_default_dns_sleep: 120
 acme_sh_default_extra_flags_issue: ""
 acme_sh_default_extra_flags_renew: ""
 acme_sh_default_extra_flags_install_cert: ""
+acme_sh_default_extra_flags_deploy_cert: ""
```

Comment:
Did you mean to name this `deploy_hook`?

Reply:
I'm not sure if we should name it `acme_sh_default_extra_flags_deploy_hook` as this is to pass extra flags to `deploy` command from Acme.sh which is deploying certificate like the`install` command does. But if you think this is better to rename I can do it.
For example a deploy command to format and copy to the right path for Haproxy : 

`acme.sh --deploy -d haproxy.example.com --deploy-hook haproxy`